src/reformat.c

```diff
@@ -313,6 +333,19 @@ avifResult avifImageRGBToYUV(avifImage * image, const avifRGBImage * rgb)
                             yuvBlock[bI][bJ].y = 0.5f * rgbPixel[1] + 0.25f * (rgbPixel[0] + rgbPixel[2]);
                             yuvBlock[bI][bJ].u = 0.5f * rgbPixel[1] - 0.25f * (rgbPixel[0] + rgbPixel[2]);
                             yuvBlock[bI][bJ].v = 0.5f * (rgbPixel[0] - rgbPixel[2]);
+#if defined(AVIF_ENABLE_EXPERIMENTAL_YCGCO_R)
+                        } else if (state.mode == AVIF_REFORMAT_MODE_YCGCO_RE || state.mode == AVIF_REFORMAT_MODE_YCGCO_RO) {
+                            // Formulas from JVET-U0093.
+                            const int R = avifRoundf(rgbPixel[0] * rgbMaxChannelF);
+                            const int G = avifRoundf(rgbPixel[1] * rgbMaxChannelF);
+                            const int B = avifRoundf(rgbPixel[2] * rgbMaxChannelF);
```

Comment:
In lines 339-341, should we apply Clip3() before avifRoundf()?

See Formulae (31) to (33) in ISO/IEC DIS 23091-2:2022(E).

Reply:
Done.